hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/impl/CryptoPreTransactionHandlerImpl.java

```diff
@@ -44,12 +44,12 @@ public CryptoPreTransactionHandlerImpl(@Nonnull final AccountStore accountStore)
 
     @Override
     /** {@inheritDoc} */
-    public TransactionMetadata preHandleCryptoCreate(final TransactionBody tx) {
-        final var op = tx.getCryptoCreateAccount();
+    public TransactionMetadata preHandleCryptoCreate(final TransactionBody txn) {
+        final var op = txn.getCryptoCreateAccount();
         final var key = asHederaKey(op.getKey());
         final var receiverSigReq = op.getReceiverSigRequired();
-        final var payer = tx.getTransactionID().getAccountID();
-        return createAccountSigningMetadata(tx, key, receiverSigReq, payer);
+        final var payer = txn.getTransactionID().getAccountID();
```

Comment:
Can txn.getTransactionID() be null?

Reply:
It cannot be, it explicitly checks for null and returns default instance if true.